README.md

```diff
@@ -84,6 +99,27 @@ Is a list of remote clusters that may define the following:
 }
 ```
 
+`cluster2` configuration:
+
+```
+{
+  "local": {
+    "name": "cluster2"
+  },
+  "remotes": [
+    {
+      "name": "cluster1",
```

Comment:
Using name as cluster1 fails to netlink dependency.
Would suggest either to replace cluster1 with a name which of len 5 or add a comment mentioning the length of the name should not exceed 5.

Reply:
hmm you are right, it would be better if that is a name that can actually be used by someone copying the config without throwing errors.